concepts/05 Widgets/DataGrid/00 Getting Started with DataGrid/80 Export Data to Excel.md

```diff
@@ -0,0 +1,216 @@
+            </DxDataGrid>
+        </div>
+    </template>
+
+    <script>
+    import {
+        DxDataGrid,
+        // ...
+        DxExport
+    } from 'devextreme-vue/data-grid';
+    import { Workbook } from 'exceljs';
+    import saveAs from 'file-saver';
+    import { exportDataGrid } from 'devextreme/excel_exporter';
+
+    export default {
+        components: {
+            DxDataGrid,
+            // ...
+            DxExport
+        },
+        // ...
+        methods: {
+            exportGrid(e) {
+                const workbook = new Workbook(); 
+                const worksheet = workbook.addWorksheet("Main sheet"); 
+                exportDataGrid({ 
+                    worksheet: worksheet, 
+                    component: e.component
+                }).then(function() {
+                    workbook.xlsx.writeBuffer().then(function(buffer) { 
+                        saveAs(new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx"); 
+                    }); 
+                }); 
+                e.cancel = true; 
+            }
+        }
+    }
+    </script>
+
+    <style>
+    /* ... */
+    </style>
+
+##### React
+
+    <!-- tab: Installation command -->
+    npm install --save exceljs file-saver
+
+    <!-- tab: App.js -->
+    import React, { useState } from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+    import './App.css';
+
+    import {
+        DataGrid,
+        Column,
+        // ...
+        Export
+    } from 'devextreme-react/data-grid';
+
+    import { Workbook } from 'exceljs';
+    import saveAs from 'file-saver';
+    import { exportDataGrid } from 'devextreme/excel_exporter';
+    // ...
+
+    function exportGrid(e) {
+        const workbook = new Workbook(); 
+        const worksheet = workbook.addWorksheet("Main sheet"); 
+        exportDataGrid({ 
+            worksheet: worksheet, 
+            component: e.component
+        }).then(function() {
+            workbook.xlsx.writeBuffer().then(function(buffer) { 
+                saveAs(new Blob([buffer], { type: "application/octet-stream" }), "DataGrid.xlsx"); 
+            }); 
+        });
+        e.cancel = true; 
+    }
+
+    function App() {
+        // ...
+        return (
+            <div className="App">
+                <DataGrid ...
+                    onExporting={exportGrid}>
+                    {/* ... */}
+                    <Export enabled={true} />
+                </DataGrid>
+            </div>
+        );
+    }
+
+    export default App;
+    
+---
+
+Run the code and click the Export button in the upper-right corner. Grid data should be exported to the DataGrid.xlsx file.
+
+The **exportDataGrid(options)** method can be called at any point in your application, not necessarily using the **DataGrid**'s export UI. For example, you can implement a standalone button and call this method when the button is clicked.
```

Comment:
```suggestion
You can call the **exportDataGrid(options)** method at any point in your application, not necessarily in the **DataGrid**'s export UI. For example, you can implement a standalone button and call this method when the button is clicked.
```

Reply:
You (the developer) can't call a method in the UI, users can. That's why I decided to go with passive voice here.

Reply:
Also reasonable